cogboard-webapp/src/components/UserLogin.js

```diff
@@ -86,6 +88,7 @@ const UserLogin = () => {
             margin="normal"
             value={values.username}
             onKeyPress={(ev) => handleLoginOnEnterPress(ev, values)}
+            inputProps={{'data-cy': 'user-login-username-input'}}
```

Comment:
Better will be login-form-username input (same situation with password, submit and validations in this form)